tests/core/process.c

```diff
@@ -293,6 +302,128 @@ TESTCASE (linux_process_modules)
   dlclose (lib);
 }
 
+TESTCASE (linux_get_cpu_from_auxv_null_32bit)
+{
+  GumCpuType cpu32;
+#if defined (HAVE_I386)
+  cpu32 = GUM_CPU_IA32;
+#elif defined (HAVE_ARM) || defined (HAVE_ARM64)
+  cpu32 = GUM_CPU_ARM;
+#elif defined (HAVE_MIPS)
+  cpu32 = GUM_CPU_MIPS;
+#else
+# error Unsupported architecture
+#endif
+  GumCpuType result;
+  guint32 test_data[] =
+  {
+    AT_NULL, 0
+  };
+  result = gum_linux_cpu_type_from_auxv ((guint8 *)test_data,
```

Comment:
The `result` is only used once and doesn't add any clarity, so perhaps the statement should just be inlined into the assertion?